src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,69 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
 
     public static void main(String[] args) {
+        text();
+        boolean param = true;
+        while (param) {
+            Scanner sc = new Scanner(System.in);
```

Comment:
Move scanner creation out of the loop. It's not needed to create it in each iteration.